scripts/deploy-federation.sh

```diff
@@ -50,8 +50,10 @@ set -o pipefail
 source "$(dirname "${BASH_SOURCE}")/util.sh"
 
 function deploy-with-helm() {
-  # RBAC should be enabled to avoid CI fail because CI K8s uses RBAC for Tiller
-  cat <<EOF | kubectl apply -f -
+  # Don't install tiller if we already have a working install.
+  if ! helm version --server; then
```

Comment:
This is correct from logic to have `if` for helm initialization. But it shows an error message `Error: could not find tiller`. It is not an error condition here. So, it should be better to clean this message to avoid misunderstanding of the message.

Reply:
I just removed the error output, as it will be displayed lower down.